TestBenches/TrackletProcessor_test.cpp

```diff
@@ -168,144 +166,6 @@ std::cout<<module_name[MODULE_];
-        err += compareMemWithFile<TrackletProjectionMemory<BARREL2S> >(tproj_barrel_2s[TP::L4PHIB], fout, ievt,
-                                                       "\nTrackletProjection (L4PHIB)", truncation);
-      else if (tproj_name.find("_L4PHIC") != string::npos)
-        err += compareMemWithFile<TrackletProjectionMemory<BARREL2S> >(tproj_barrel_2s[TP::L4PHIC], fout, ievt,
-                                                       "\nTrackletProjection (L4PHIC)", truncation);
-      else if (tproj_name.find("_L4PHID") != string::npos)
-        err += compareMemWithFile<TrackletProjectionMemory<BARREL2S> >(tproj_barrel_2s[TP::L4PHID], fout, ievt,
-                                                       "\nTrackletProjection (L4PHID)", truncation);
-      else if (tproj_name.find("_L5PHIA") != string::npos)
-        err += compareMemWithFile<TrackletProjectionMemory<BARREL2S> >(tproj_barrel_2s[TP::L5PHIA], fout, ievt,
-                                                       "\nTrackletProjection (L5PHIA)", truncation);
-      else if (tproj_name.find("_L5PHIB") != string::npos)
-        err += compareMemWithFile<TrackletProjectionMemory<BARREL2S> >(tproj_barrel_2s[TP::L5PHIB], fout, ievt,
-                                                       "\nTrackletProjection (L5PHIB)", truncation);
-      else if (tproj_name.find("_L5PHIC") != string::npos)
-        err += compareMemWithFile<TrackletProjectionMemory<BARREL2S> >(tproj_barrel_2s[TP::L5PHIC], fout, ievt,
-                                                       "\nTrackletProjection (L5PHIC)", truncation);
-      else if (tproj_name.find("_L5PHID") != string::npos)
-        err += compareMemWithFile<TrackletProjectionMemory<BARREL2S> >(tproj_barrel_2s[TP::L5PHID], fout, ievt,
-                                                       "\nTrackletProjection (L5PHID)", truncation);
-      else if (tproj_name.find("_L6PHIA") != string::npos)
-        err += compareMemWithFile<TrackletProjectionMemory<BARREL2S> >(tproj_barrel_2s[TP::L6PHIA], fout, ievt,
-                                                       "\nTrackletProjection (L6PHIA)", truncation);
-      else if (tproj_name.find("_L6PHIB") != string::npos)
-        err += compareMemWithFile<TrackletProjectionMemory<BARREL2S> >(tproj_barrel_2s[TP::L6PHIB], fout, ievt,
-                                                       "\nTrackletProjection (L6PHIB)", truncation);
-      else if (tproj_name.find("_L6PHIC") != string::npos)
-        err += compareMemWithFile<TrackletProjectionMemory<BARREL2S> >(tproj_barrel_2s[TP::L6PHIC], fout, ievt,
-                                                       "\nTrackletProjection (L6PHIC)", truncation);
-      else if (tproj_name.find("_L6PHID") != string::npos)
-        err += compareMemWithFile<TrackletProjectionMemory<BARREL2S> >(tproj_barrel_2s[TP::L6PHID], fout, ievt,
-                                                       "\nTrackletProjection (L6PHID)", truncation);
-
-    }
```

Comment:
@aryd I guess you removed these comparisons because the test vectors no longer agree with the HLS. I know we don't care about this on the FW side, but will it be an issue for the future SW emulation (@tomalin)?

Reply:
BTW, in either case, this question relates more to the legacy emulation in CMSSW, so it need not delay merging this PR.

Reply:
Though we have the code still in HLS to calculate the projections in the TP modules the (default) configuration does not produce these memories as they are not used by any other module. Hence I removed them in this test...
But as I said the code is still there to calculate the projection if you have the right configuration. It is clear that it is going to be harder and harder to maintain the future SW emulation if we don't migrate to the PC and VMSMER modules...

Reply:
The Future SW is not reliant on the HLS test-benches. But I guess if you're no longer validating the content of the TrackletProjectionMemories, they will start to deviate from emulation, so it may become problematic for the Future SW in time. But hopefully, (if Future SW remains alive), it will migrate to 2-FPGA at some point.